miniupnpd/genconfig.sh

```diff
@@ -233,6 +233,7 @@ case $OS_NAME in
 			FW=pf
 		fi
 		echo "#define USE_IFACEWATCHER 1" >> ${CONFIGFILE}
+		echo "#define PFRULE_INOUT_COUNTS" >> ${CONFIGFILE}
```

Comment:
I could, but DragonFly BSD 2.8 was released almost 6 years ago, and I can't really check if miniupnpd builds or run properly on such an old system.